### DISCUSSION TITLE
Single coil emulated data from multi coil brain dataset

### DISCUSSION THREAD

**Discussion by jainspoornima** [facebookresearch/fastMRI#201] [Q&A] [answered]
I wished to ask, has anyone created the emulated single coil dataset from the multicoil brain dataset?

**Comment by mmuckley** [comment]
Converting to a discussion as this does not document a bug.

@jainspoornima, to my knowledge no one has done this.

**Comment by hellopipu** [comment, marked as answer]
Hi @jainspoornima , I just found this question, maybe too late for you. This [paper](https://msp.org/camcos/2020/15-2/p01.xhtml) describes the method to emulate single-coil knee from multi-coil knee. Following this paper, I created the emulated single coil dataset from a multicoil cardiac dataset for my own project months ago , you can take my code as a reference [here](https://github.com/hellopipu/HQS-Net/blob/665f7a0773b3184de0afbad2765ca9e256b955ea/preprocess_ocmr.py#L65) if you want to generate single-coil brain data by yourself.
